Cyclic reference error

Hi, encountered the following error while building Ginco:

```
mvn clean package
[INFO] Scanning for projects...
[ERROR] The projects in the reactor contain a cyclic reference: Edge between 'Vertex{label='fr.smile.mcc.ginco:ginco-quality-tool-conf:2.0.1'}' and 'Vertex{label='fr.smile.mcc.ginco:ginco:2.0.1'}' introduces to cycle in the graph fr.smile.mcc.ginco:ginco:2.0.1 --> fr.smile.mcc.ginco:ginco-quality-tool-conf:2.0.1 --> fr.smile.mcc.ginco:ginco:2.0.1 -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/ProjectCycleException
```

Running Maven with -X option I got:

```
Apache Maven 3.2.3 (33f8c3e1027c3ddde99d3cdebad2656a31e8fdf4; 2014-08-11T22:58:10+02:00)
Maven home: C:\tools\apache-maven\bin\..
Java version: 1.8.0_31, vendor: Oracle Corporation
Java home: C:\Program Files\Java\jdk1.8.0_31\jre
Default locale: de_DE, platform encoding: Cp1252
OS name: "windows 7", version: "6.1", arch: "amd64", family: "dos"
[INFO] Error stacktraces are turned on.
[DEBUG] Reading global settings from C:\tools\apache-maven\bin\..\conf\settings.xml
[DEBUG] Reading user settings from C:\Users\user123\.m2\settings.xml
[DEBUG] Using local repository at C:\Users\user123\.m2\repository
[DEBUG] Using manager EnhancedLocalRepositoryManager with priority 10.0 for C:\Users\user123\.m2\repository
[INFO] Scanning for projects...
[DEBUG] Extension realms for project fr.smile.mcc.ginco:ginco:pom:2.0.1: (none)
[DEBUG] Looking up lifecyle mappings for packaging pom from ClassRealm[plexus.core, parent: null]
[DEBUG] Extension realms for project fr.smile.mcc.ginco:ginco-quality-tool-conf:jar:2.0.1: (none)
[DEBUG] Looking up lifecyle mappings for packaging jar from ClassRealm[plexus.core, parent: null]
[DEBUG] Extension realms for project fr.smile.mcc.ginco:ginco-api:jar:2.0.1: (none)
[DEBUG] Looking up lifecyle mappings for packaging jar from ClassRealm[plexus.core, parent: null]
[DEBUG] Extension realms for project fr.smile.mcc.ginco:ginco-commons:jar:2.0.1: (none)
[DEBUG] Looking up lifecyle mappings for packaging jar from ClassRealm[plexus.core, parent: null]
[DEBUG] Extension realms for project fr.smile.mcc.ginco:ginco-impl:jar:2.0.1: (none)
[DEBUG] Looking up lifecyle mappings for packaging jar from ClassRealm[plexus.core, parent: null]
[DEBUG] Extension realms for project fr.smile.mcc.ginco:ginco-admin:war:2.0.1: (none)
[DEBUG] Looking up lifecyle mappings for packaging war from ClassRealm[plexus.core, parent: null]
[DEBUG] Extension realms for project fr.smile.mcc.ginco:ginco-webservices:war:2.0.1: (none)
[DEBUG] Looking up lifecyle mappings for packaging war from ClassRealm[plexus.core, parent: null]
[DEBUG] Extension realms for project fr.smile.mcc.ginco:ginco-solr-conf:jar:2.0.1: (none)
[DEBUG] Looking up lifecyle mappings for packaging jar from ClassRealm[plexus.core, parent: null]
[DEBUG] Extension realms for project fr.smile.mcc.ginco:ginco-cli:jar:2.0.1: (none)
[DEBUG] Looking up lifecyle mappings for packaging jar from ClassRealm[plexus.core, parent: null]
[ERROR] The projects in the reactor contain a cyclic reference: Edge between 'Vertex{label='fr.smile.mcc.ginco:ginco-quality-tool-conf:2.0.1'}' and 'Vertex{label='fr.smile.mcc.ginco:ginco:2.0.1'}' introduces to cycle in the graph fr.smile.mcc.ginco:ginco:2.0.1 --> fr.smile.mcc.ginco:ginco-quality-tool-conf:2.0.1 --> fr.smile.mcc.ginco:ginco:2.0.1 -> [Help 1]
org.apache.maven.ProjectCycleException: The projects in the reactor contain a cyclic reference: Edge between 'Vertex{label='fr.smile.mcc.ginco:ginco-quality-tool-conf:2.0.1'}' and 'Vertex{label='fr.smile.mcc.ginco:ginco:2.0.1'}' introduces to cycle in the graph fr.smile.mcc.ginco:ginco:2.0.1 --> fr.smile.mcc.ginco:ginco-quality-tool-conf:2.0.1 --> fr.smile.mcc.ginco:ginco:2.0.1
	at org.apache.maven.DefaultMaven.createProjectDependencyGraph(DefaultMaven.java:826)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:330)
	at org.apache.maven.DefaultMaven.execute(DefaultMaven.java:154)
	at org.apache.maven.cli.MavenCli.execute(MavenCli.java:582)
	at org.apache.maven.cli.MavenCli.doMain(MavenCli.java:214)
	at org.apache.maven.cli.MavenCli.main(MavenCli.java:158)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced(Launcher.java:289)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launch(Launcher.java:229)
	at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode(Launcher.java:415)
	at org.codehaus.plexus.classworlds.launcher.Launcher.main(Launcher.java:356)
Caused by: org.codehaus.plexus.util.dag.CycleDetectedException: Edge between 'Vertex{label='fr.smile.mcc.ginco:ginco-quality-tool-conf:2.0.1'}' and 'Vertex{label='fr.smile.mcc.ginco:ginco:2.0.1'}' introduces to cycle in the graph fr.smile.mcc.ginco:ginco:2.0.1 --> fr.smile.mcc.ginco:ginco-quality-tool-conf:2.0.1 --> fr.smile.mcc.ginco:ginco:2.0.1
	at org.codehaus.plexus.util.dag.DAG.addEdge(DAG.java:141)
	at org.apache.maven.project.ProjectSorter.addEdge(ProjectSorter.java:220)
	at org.apache.maven.project.ProjectSorter.addEdge(ProjectSorter.java:185)
	at org.apache.maven.project.ProjectSorter.<init>(ProjectSorter.java:126)
	at org.apache.maven.DefaultProjectDependencyGraph.<init>(DefaultProjectDependencyGraph.java:53)
	at org.apache.maven.DefaultMaven.createProjectDependencyGraph(DefaultMaven.java:805)
	... 13 more
[ERROR] 
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/ProjectCycleException
```

Ok, problem is reproduced with latest maven
hufon@STJean:/tmp/ginco$ ~/javalibs/apache-maven-3.3.3/bin/mvn clean package
[INFO] Scanning for projects...
[ERROR] [ERROR] The projects in the reactor contain a cyclic reference: Edge between 'Vertex{label='fr.smile.mcc.ginco:ginco-quality-tool-conf:2.0.1'}' and 'Vertex{label='fr.smile.mcc.ginco:ginco:2.0.1'}' introduces to cycle in the graph fr.smile.mcc.ginco:ginco:2.0.1 --> fr.smile.mcc.ginco:ginco-quality-tool-conf:2.0.1 --> fr.smile.mcc.ginco:ginco:2.0.1 @ 
[ERROR] The projects in the reactor contain a cyclic reference: Edge between 'Vertex{label='fr.smile.mcc.ginco:ginco-quality-tool-conf:2.0.1'}' and 'Vertex{label='fr.smile.mcc.ginco:ginco:2.0.1'}' introduces to cycle in the graph fr.smile.mcc.ginco:ginco:2.0.1 --> fr.smile.mcc.ginco:ginco-quality-tool-conf:2.0.1 --> fr.smile.mcc.ginco:ginco:2.0.1 -> [Help 1]

--- a/ginco-admin/src/main/webapp/app/locale/fr/view/BottomFormToolbar.js
+++ b/ginco-admin/src/main/webapp/app/locale/fr/view/BottomFormToolbar.js
@@ -1,4 +1,4 @@
-﻿/**
+/**
  * Copyright or © or Copr. Ministère Français chargé de la Culture
  * et de la Communication (2013)
  * <p/>

--- a/ginco-admin/src/main/webapp/app/locale/fr/view/TopToolbar.js
+++ b/ginco-admin/src/main/webapp/app/locale/fr/view/TopToolbar.js
@@ -1,4 +1,4 @@
-﻿/**
+/**
  * Copyright or © or Copr. Ministère Français chargé de la Culture
  * et de la Communication (2013)
  * <p/>

